builtin/game/features.lua

```diff
@@ -29,6 +29,7 @@ core.features = {
 	compress_zstd = true,
 	sound_params_start_time = true,
 	physics_overrides_v2 = true,
+	lightIntensity = true,
```

Comment:
Please follow the `snake_case` naming convention for everything Lua (that isn't a "class").

Reply:
fixed